delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewFields.scala

```diff
@@ -28,8 +27,8 @@ import scala.concurrent.duration.FiniteDuration
 final case class CompositeViewFields(
     name: Option[String],
     description: Option[String],
-    sources: NonEmptySet[CompositeViewSourceFields],
-    projections: NonEmptySet[CompositeViewProjectionFields],
+    sources: NonEmptyList[CompositeViewSourceFields],
```

Comment:
Same, here, we check for duplication before creating the composite view anyway

Reply:
Is this an advantage? Set is telling us that ordering doesn't matter and the duplicates don't have an effect, which seems good?

Reply:
Yes because `NonEmptyList` is a lot easier to use.
`NonEmptySet` is based on a sorted set, requires an order and did not bring value here
as there is also a check for duplicates on the id of sources/projections when evaluating as sources and projections must all have different ids.